ecs-cli/modules/cli/compose/containerconfig/containerconfig.go

```diff
@@ -3,14 +3,15 @@ package containerconfig
 import "github.com/aws/aws-sdk-go/service/ecs"
 
 // ContainerConfig all compose fields supported by the ecs-cli
-// TODO: finalize fields
+// TODO: finalize fields (Note: Devices and HealthCheck not currently supported)
 type ContainerConfig struct {
 	Name string
 
 	CapAdd                []string
 	CapDrop               []string
 	Command               []string
-	Devices               []string
+	CPUShares             int64
```

Comment:
should we just call this "CPU", per the name of the task def field?

Reply:
I wasn't sure if we wanted to err more on the side of naming things in this struct more closely with ECS Container Definition fields or with Docker Compose fields. Could go either way!

Reply:
ah ok, i thought we agreed - for v3 you asked I change "LogConfiguration" to align with ECS structs; I do think that makes more sense, since their final destination is a task def and the diff versions of Docker don't always agree on what to call something (see volumes_from, log_driver, etc.)